cypress/support/cypress-support/src/main.js

```diff
@@ -127,12 +127,21 @@ export default function (module) {
                           externalData.scenarioNames[module][scenario][key];
                       });
                     }
-                    Cypress.env(CONSTANTS.MODULEREQIDJSON, fcsData);
                   });
-                } else {
-                  Cypress.env(CONSTANTS.MODULEREQIDJSON, fcsData);
+                  configModulesData.map((scenario) => {
+                    if (!fcsModulesData.includes(scenario)) {
+                      fcsData.scenarioNames[module][scenario] =
+                        externalData.scenarioNames[module][scenario];
+                    }
+                  });
+                }
+              });
+              config.map((module) => {
+                if (!FCS.includes(module)) {
+                  fcsData.scenarioNames[module] = externalData.scenarioNames[module];
```

Comment:
what if individual test cases inside a module is missing in FCS ? 
Eg, both fcs and config module has module name as keyboard. 
FCS and config Module has test case 1
FCS doesnt have Test case 2, it is only present in config Module


It should handle all below scenarios as mentioned in 1945
Also, what will happen in case of whole module

Reply:
From line no 131 to 136 is to handle a particular testcase is missing inside a module that is present in config but not in FCS. 
From line 141 to 144 is to handle whole module being missing.

Reply:
Tested the individual testcase missing by removing this scenario `Accessibility.closedCaptionsSettings - Positive Scenario: Disable closedcaptions` in FCS moduleReqID and placed the beforeOperation only in configModule.

Report: https://reporting.firecert.comcast.com/cucumberReports/0529-4b32-cd48-9a4b/index.html

Reply:
Did we test all below scenarios as mentioned in 1945 table ?> 
also in the above report, launch is happening multiple times instead of just once. why is that ?>